README.md

```diff
@@ -139,6 +139,36 @@ sudo dnf install glfw glfw-devel freetype-devel
 
 glfw dependency will be removed soon.
 
+
+## Troubleshooting:
+
+You can see how v invokes the C backend compiler with `v -show_c_cmd source.v` .
+
+You can produce a .c file, *without* compiling it further with `v -o file.c file.v` . 
```

Comment:
Would be nice to slightly improve the integrated help (invoked by `v -h`) to tell which file extensions are supported and what they correspondingly produce. Currently this is not at all indicative and one expects `-o` to produce a platform-dependent binary and nothing else.

Reply:
I agree that improving the -h description is also good. 
That may be done in another PR.

About -o x.c producing a c file, I think it is intuitive enough. 
The alternative is to add a separate flag.

Reply:
See #1926 .